MovieTime/MovieTime/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
how did you choose your app icon and splash screen? are you the author of these images?

Reply:
Since I was creating a Movie app and I needed to decide the application’s UI/UX, I searched for a royalty-free icon.
For personal projects I usually follow this idea or create my own icons.
For large projects normally the layout is predefined, if something is still missing I follow the idea of Royalty-free icons.